apps/docs-site/src/css/custom.css

```
@@ -575,8 +650,71 @@ span.kx-live-title,
+ * Minimal preflight reset, scoped to `.tw-enable`.
+ *
+ * `tailwind-config.js` sets `corePlugins.preflight: false` so the
+ * Play CDN doesn't reset Docusaurus's chrome. The cost is that
+ * default user-agent `<button>` styling (outset border, system
+ * background, fixed appearance) leaks through everywhere Tailwind
+ * is applied — that's why the recipe's day cells looked like 90s
+ * 3D buttons even though the Tailwind classes were correct.
+ *
+ * Re-add only the rules the calendar examples need, locked behind
+ * `.tw-enable` so they don't escape the recipe pages. See issue #35.
  * ---------------------------------------------------------*/
```

Stylelint (17.9.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)